ext/openssl/tests/openssl-get-cipher-methods.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+array openssl_get_cipher_methods ([ bool $aliases = false ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php 
+if (!extension_loaded("openssl")) print "skip";
+if (OPENSSL_VERSION_NUMBER < 0x10000000) die("skip Output requires OpenSSL 1.0");
+?>
+--FILE--
+<?php
+$getCipherTrue = openssl_get_cipher_methods(true);
+$getCipherFalse = openssl_get_cipher_methods(false);
+
+if ((is_array($getCipherTrue)) && (!is_null($getCipherTrue))) {
```

Comment:
the second part of condition (!is_null) is useless as if it's array, then it can't be null...